src/picker/typer/cliclick.py

```diff
@@ -0,0 +1,36 @@
+from subprocess import run
+from typing import List
+
+from ..abstractionhelper import is_installed, is_wayland, is_macos
+from .typer import Typer
+
+
+class CliclickTyper(Typer):
+    @staticmethod
+    def supported() -> bool:
+        return is_installed("cliclick") and is_macos()
+
+    @staticmethod
+    def name() -> str:
+        return "cliclick"
+
+    def get_active_window(self) -> str:
+        return ""
+
+    def type_characters(self, characters: str, active_window: str) -> None:
+        pass
+
+
+    def insert_from_clipboard(self, active_window: str) -> None:
+        run(
+            [
+                "cliclick",
+                "-w 100",
+                "kd:cmd",
```

Comment:
For Linux, it uses `Alt` + `Insert`. Is there a difference between both on Mac?

Reply:
I'm clearly not a Mac expert (using it for just 6 months 😬 ), but as far as I can see there is no Insert key on Mac keyboard and there is no equivalent in my knowledge.
cmd+c and cmd+v (what I implemented) seems good enough as they work in all windows to copy/paste data from the general purpose clipboard. This clipboarder works as expected in rofimoji so I did not look further.

I guess alt+insert was used to avoid the struggle where some apps could use ctrl+v and others (like terms) would have required ctrl+shift+v ?